ocaml/tests/test_basic.ml

```diff
@@ -27,19 +23,7 @@ let test_mock_db () =
   ignore (Db.Blob.get_record ~__context ~self:blob_ref) ;
   ignore (Db.VM.get_all_records ~__context) ;
   let blob_name = Db.Blob.get_name_label ~__context ~self:blob_ref in
-  assert_equal blob_name "BLOB"
-
-let test_assert_licensed_storage_motion () = skip "TODO" ;
-  let __context = Mock.make_context_with_new_db "Mock context" in
-  let licensed = try Xapi_vm_migrate.assert_licensed_storage_motion ~__context; true
-    with _ -> false in
-  assert_bool "Not licensed for SXM" licensed
```

Comment:
Is this test no longer valid or is this function tested elsewhere?

Reply:
I decided to simply delete this because it was not run: at line 32, we have `skip "TODO"`

Reply:
Is it worth removing the `skip "TODO"` so it runs and including it?

Reply:
I don't know whether it works or not, but probably not because it was skipped. My thinking was that it's probably not worth trying to debug old legacy tests :thinking:

Reply:
FYI all the tests in Test_basic are really just demo tests that were written when unit tests first became a thing - they're not particularly useful as actual tests of the xapi codebase.

Reply:
Oh ok, thanks a lot for the info @johnelse  - then I'll just remove it completely :smile: